lvm2/pkg.yaml

```diff
@@ -4,7 +4,7 @@ shell: /toolchain/bin/bash
 dependencies:
   - stage: base
   - stage: libaio
-  - stage: eudev
+  - stage: systemd-udevd
```

Comment:
```
4.868 configure: error: Package requirements (libudev >= 143) were not met:
4.868 
4.868 No package 'libudev' found
4.868 
4.868 Consider adjusting the PKG_CONFIG_PATH environment variable if you
4.868 installed software in a non-standard prefix.
4.868 
4.868 Alternatively, you may set the environment variables UDEV_CFLAGS
4.868 and UDEV_LIBS to avoid the need to call pkg-config.
4.868 See the pkg-config man page for more details.
```

Reply:
Fixed, but something else regressed

Reply:
Yes, it's not just me: https://github.com/siderolabs/pkgs/actions/runs/11091800704/job/30816036407?pr=1038#step:9:5646

Reply:
Fixing the thin archive issue